TESTS/mbed-crypto/sanity/main.cpp

```diff
@@ -125,6 +125,34 @@ void test_crypto_symmetric_cipher_encrypt_decrypt(void)
     mbedtls_psa_crypto_free();
 }
 
+void test_crypto_asymmetric_sign_verify(void)
+{
+    psa_key_slot_t slot = 1;
+    psa_key_type_t key_type = PSA_KEY_TYPE_RSA_KEYPAIR;
+    psa_algorithm_t alg = PSA_ALG_RSA_PKCS1V15_CRYPT;
+    size_t key_bits = 512;
+    psa_key_policy_t policy;
+    unsigned char input[100];
```

Comment:
Could we test with a fixed input so we can check the result of the signature matches what we expect? Note that this would only work for deterministic signature algorithms.

Reply:
Same comment as before, the key is generated.

Reply:
The signature test doesn't need to use a generated key.

Reply:
Ok I'll change this to use a fixed imported key then.